ci-operator/step-registry/ipi/conf/azurestack/creds/ipi-conf-azurestack-creds-commands.sh

```diff
@@ -62,7 +73,7 @@ do
       continue
   fi
 
-  filename=manifest_${SECRET_NAMESPACE}_secret.yml
+  filename=manifest_${SECRET_NAME}_secret.yml
```

Comment:
In bootstrap gather logs, saw the following error:
```
Dec 14 06:08:37 ci-op-9hsh6104-0a8e3-pgxsd-master-2 kubenswrapper[2302]: E1214 06:08:37.021412    2302 pod_workers.go:965] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"azure-inject-credentials\" with CreateContainerConfigError: \"secret \\\"azure-cloud-credentials\\\" not found\"" pod="openshift-cloud-controller-manager/azure-cloud-controller-manager-5d9db6c477-nd98g" podUID=1b9c3965-db4c-4906-829a-095af023c5fe
```
So the change should be `filename=manifest_${SECRET_NAMESPACE}_${SECRET_NAME}_secret.yml`

Reply:
renamed filename as you suggested, it works. Installation is passed.